### DISCUSSION TITLE
Multiple simultanous vpn connections

### DISCUSSION THREAD

**Discussion by tibz7** [strongswan/strongswan#1812] [Q&A]
Hello, 

How is it possible to use two different vpns (2 diffrent gateways) simultanously on linux?
On OSX there was no problems, but on linux (fedora 38/KDE) everytime i try to start two vpns via the network manager UI I'm getting "The 'strongswan' plugin only supports a single active connection. "

**Comment by tobiasbrunner** [comment]
This is a duplicate of #1493.
